prow/config/jobs/istio.yaml

```diff
@@ -76,6 +76,21 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: --istio.test.skipVM
 
+  - name: integ-ambient-calico
+    modifiers: [presubmit_optional]
```

Comment:
I don't think we should run this on presubmit. Its a useful test but IMO we don't want to bloat up our test matrix this much (esp since we will be adding cilium, ..., in the future). can we make it `[presubmit_optional, presubmit_skipped]`?

Reply:
makes sense. thanks for the suggestion